### DISCUSSION TITLE
Cannot open the app

### DISCUSSION THREAD

**Discussion by MRSalguod** [gorilla-devs/ferium#384] [Q&A] [answered]
How do I install this. every time i run ferium.exe it flashes a black screen then closes. am i doing something wrong?

**Comment by MRSalguod** [comment]
figured it out.

**Reply by Benzxlol-w** [reply]
how did you do it?
I have the same ossie

**Reply by Benzxlol-w** [reply]
Issue*

**Comment by theRookieCoder** [comment, marked as answer]
Ferium is a CLI application, it's meant to be used in a terminal or command prompt.

**It does not have a GUI**.

**Reply by theRookieCoder** [reply]
@Benzxlol-w
